rails/app/controllers/api/v1/jwt_controller.rb

```diff
@@ -149,7 +149,22 @@ def portal
     user, learner, teacher = handle_initial_auth
 
     claims = {}
-    if learner
+    if params[:researcher] == "true"
+      # Note: no check is done to see if the user is a researcher for any projects.
+      # The researcher user_type is used only by clients to know what type of JWT they have
+      # and all requests to the portal using the JWT are checked for permissions using the user_id
+      # which will check if the user has permissions to access the data.
+      # These JWTs are used currently only by generated links in reports and those links may
+      # be used by people who are not researchers but have access to the data as project admins
```

Comment:
This seems unsafe, since some code could start relying on this researcher: true claim. But I guess if hard to implement then it isn't worth it.